packages/preferences/constants.js

```diff
@@ -1,6 +1,6 @@
 const APPS_EXTRAS = 'appsandextras';
 const SECURITY = 'security';
-const GENERAL = 'general';
+const GENERAL = 'general-settings';
```

Comment:
Oops! This is related to my earlier comment about the similar looking but very different preferences package. You'll want to revert this one back to the original, which should help in making preferences accessible again. 😺 

```suggestion
const GENERAL = 'general';
```